content/docs/csidriver/installation/operator/_index.md

```diff
@@ -413,7 +405,7 @@ spec:
 You can set the field ***replicas*** to a higher number than `1` for the latest driver versions.
 
 Note - The `image` field should point to the correct image tag for version of the driver you are installing.  
-For e.g. - If you wish to install v1.4 of the CSI PowerMax driver, use the image tag `dellemc/csi-powermax:v1.4.0.000R`
+For e.g. - If you wish to install v1.10.0 of the CSI PowerMax driver, use the image tag `dellemc/csi-powermax:v1.10.0`
```

Comment:
Change the version to v2.5.0 in both places

Reply:
Done